[docs]: Update EAS Update getting started (#19182)
expo/expo: docs/pages/eas-update/getting-started.md

Create a project by running:

> <Terminal cmd={['$ npx create-expo-app && ']} />

Reviewer: ```suggestion
<Terminal cmd={['$ npx create-expo-app']} />
```

Looks like unintended change.

Author: Yes it is. Thanks for catching this!

## Configure your project

To configure your project, run the following commands in the order they are specified:

<Terminal cmd={[
'# Install the latest `expo-updates` library',
'$ npx expo install expo-updates',
'',
'# Initialize your project with EAS Update',
'$ eas update:configure',
'',
'# Set up the configuration file for builds',
'$ eas build:configure',
]} cmdCopy="expo install expo-updates && eas update:configure && eas build:configure" />

> After running these commands, **eas.json** is created in the root directory of your project.

Reviewer: Something seems off to me in this rewritten sentence. What do you think about:

```suggestion
After running these commands, **eas.json** file should be present in the root directory of your project.
```

or

```suggestion
After running these commands, **eas.json** file will be created in the root directory of your project.
```
